website/docs/r/slo.html.markdown

```diff
@@ -154,6 +170,7 @@ The following arguments are supported:
   , `Other`. Defaults to `Latency`.
 - `service` - (Optional) Name of the service.
 - `application` - (Optional) Name of the application.
+- `tags` - (Optional) A map defining tag keys and tag values for the SLO.
```

Comment:
can we add the beta notice here, like we did for multi-window multi-burn

Reply:
added for `tags`, removed the same for `burn_rate`